packages/cashier/src/modules/deposit-fiat/components/deposit-fiat-iframe/deposit-fiat-iframe.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Loading } from '@deriv/components';
+import { useDepositFiatAddress } from '@deriv/hooks';
+import { observer } from '@deriv/stores';
+import './deposit-fiat-iframe.scss';
+
+const DepositFiatIframe: React.FC = observer(() => {
+    const { data: iframe_url, isSuccess } = useDepositFiatAddress();
+    const [is_loading, setIsLoading] = useState(true);
+
+    React.useEffect(() => {
+        setIsLoading(true);
+    }, [iframe_url]);
+
+    return (
+        <React.Fragment>
+            {is_loading && <Loading className='deposit-fiat-iframe__loader' is_fullscreen={false} />}
+            {isSuccess && (
+                <iframe
+                    className='deposit-fiat-iframe__iframe'
+                    onLoad={() => setIsLoading(false)}
+                    src={iframe_url}
+                    style={!is_loading ? { display: 'block' } : { display: 'none' }}
```

Comment:
```suggestion
                    style={{ display: is_loading ? 'none' : 'block' }}
```
@aum-deriv We are changing the same property conditionally so we can just change the value.